ios/brave-ios/Sources/PlaylistUI/MediaContentView.swift

```diff
@@ -0,0 +1,287 @@
+        ),
+        duration: model.duration,
+        isScrubbing: $isScrubbing
+      )
+      .tint(Color(braveSystemName: .iconInteractive))
+      .onChange(of: isScrubbing) { newValue in
+        if newValue {
+          resumePlayingAfterScrub = model.isPlaying
+          model.pause()
+        } else {
+          if resumePlayingAfterScrub {
+            model.play()
+          }
+        }
+      }
+      .task(priority: .low) {
+        self.currentTime = model.currentTime
+        for await currentTime in model.currentTimeStream {
+          self.currentTime = currentTime
+        }
+      }
+    }
+  }
+  struct PlaybackControlsView: View {
+    @ObservedObject var model: PlayerModel
+    var selectedItemTitle: String
+
+    @Environment(\.toggleFullScreen) private var toggleFullScreen
+
+    var body: some View {
+      VStack(spacing: 28) {
+        HStack(alignment: .firstTextBaseline) {
+          Text(selectedItemTitle)
+            .foregroundStyle(Color(braveSystemName: .textPrimary))
+            .font(.headline)
+            .frame(maxWidth: .infinity, alignment: .leading)
+            .multilineTextAlignment(.leading)
+            .lineLimit(1)
+          if model.allowsExternalPlayback {
+            RoutePickerView()
+          }
+        }
+        PlaybackScrubber(model: model)
+        VStack(spacing: 28) {
+          HStack {
+            Toggle(isOn: $model.isShuffleEnabled) {
+              if model.isShuffleEnabled {
+                Image(braveSystemName: "leo.shuffle.toggle-on")
+                  .transition(.opacity.animation(.linear(duration: 0.1)))
+              } else {
+                Image(braveSystemName: "leo.shuffle.off")
+                  .transition(.opacity.animation(.linear(duration: 0.1)))
+              }
+            }
+            .toggleStyle(.button)
+            Spacer()
+            Button {
+              Task { await model.seekBackwards() }
+            } label: {
+              Label("Step Back", braveSystemImage: "leo.rewind.15")
+            }
+            .buttonStyle(.playbackControl(size: .large))
+            .tint(Color(braveSystemName: .textPrimary))
+            Spacer()
+            Toggle(isOn: $model.isPlaying) {
+              if model.isPlaying {
+                Label("Pause", braveSystemImage: "leo.pause.filled")
+                  .transition(.playButtonTransition)
+              } else {
+                Label("Play", braveSystemImage: "leo.play.filled")
+                  .transition(.playButtonTransition)
+              }
+            }
+            .toggleStyle(.button)
+            .accessibilityAddTraits(!model.isPlaying ? .startsMediaSession : [])
+            .tint(Color(braveSystemName: .textPrimary))
+            .buttonStyle(.playbackControl(size: .extraLarge))
+            Spacer()
+            Button {
+              Task { await model.seekForwards() }
+            } label: {
+              Label("Step Forward", braveSystemImage: "leo.forward.15")
```

Comment:
You might wanna start adding localization slowly otherwise it will take a lot of time in the end of the project :D

Reply:
Its not actually that many strings, I've tested with compiled String Catalog test and its <20 strings total so far